skymap_scanner/server/start_scan.py

```diff
@@ -347,6 +352,7 @@ async def scan(
         output_particle_name=cfg.OUTPUT_PARTICLE_NAME,
         reco_algo=reco_algo,
         event_metadata=event_metadata,
+        realtime_format_version=realtime_format_version
```

Comment:
See comment on trailing comma--there are a few other cases where this applies but I won't add comments